packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -2271,8 +2271,6 @@ export const components = (
     },
     MuiTypography: {
       defaultProps: {
-        fontFamily:
-          "'Inter', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
```

Comment:
We don't need these? Or was this the situation it had the wrong font?

Reply:
Setting the default here unintuitively overrides the styles in `typography.ts`. They've probably been broken for awhile 😞